src/std/sxml/tal/iter.ss

```diff
@@ -0,0 +1,103 @@
+(import :std/interface :std/contract :std/iter :std/generic :std/sxml/tal/toplevel)
+(export #t)
+
```

Comment:
you probably want a (declare (fixnum)) in here.

Reply:
Where and how? Done as soon as I know. declarations are foreign to me still.

Reply:
Just put it here flat for the module.

Point being is you are doing some fixnum arithmetic with generic ops, so it is slower than it has to be; the declaration will remedy.